styleguide/source/assets/scss/01-atoms/_links.scss

```diff
@@ -1,52 +1,128 @@
 /* Links */
-a {
-  color: $topic_black;
+a.topic {
+  @include type($myriad-pro, $font-weight-regular);
+  @include font-size($h4-font-sizes);
+  color: $blue;
   text-decoration: none;
   outline: 0;
   &:hover,
   &:focus {
-    color: $topic_black;
+    color: $blue;
   }
 }
 
-.ama__link-black {
+.ama__link--white {
+  @include type($myriad-pro, $font-weight-regular);
+  @include font-size($h4-font-sizes);
+
   &:link,
   &:visited {
-    color: $topic_black;
+    color: $white;
+  }
+  &:active,
+  &:hover,
+  &:focus {
+    color: $white;
+    text-decoration: underline;
+  }
+}
+
+.ama__link--blue {
+  @include type($myriad-pro, $font-weight-regular);
+  @include font-size($h4-font-sizes);
+
+  &:link,
+  &:visited {
+    color: $blue;
   }
   // if the default link style changes this will need to include color: black; text-decoration: none;
   &:active,
   &:hover,
   &:focus {
-    color: $topic_blue;
-    text-decoration: none;
+    color: $blue;
+    text-decoration: underline;
   }
 }
 
-.ama__link-white {
+.ama__link--purple {
+  @include type($myriad-pro, $font-weight-regular);
+  @include font-size($p-font-sizes);
+
   &:link,
   &:visited {
-    color: $topic_white;
+    color: $purple;
   }
   // if the default link style changes this will need to include color: black; text-decoration: none;
   &:active,
   &:hover,
   &:focus {
-    color: $topic_white;
+    color: $purple;
     text-decoration: underline;
   }
 }
 
-.ama__link-blue {
+//Topic page links
+
+a {
+  color: $topic_black;
+  text-decoration: none;
+  outline: 0;
+  &:hover,
+  &:focus {
+    color: $topic_black;
+  }
+
+  svg {
+    height: 22px;
+    width: 22px;
+    vertical-align: middle;
+  }
+}
+
+.topic__link--black {
   &:link,
   &:visited {
-    color: $topic_blue;
+    color: $topic_black;
   }
   // if the default link style changes this will need to include color: black; text-decoration: none;
   &:active,
   &:hover,
   &:focus {
     color: $topic_blue;
-    text-decoration: underline;
+    text-decoration: none;
+  }
+}
+
+.ama__link--white,
+%link--white {
+  .topic__link--white {
+    &:link,
+    &:visited {
+      color: $topic_white;
+    }
+    // if the default link style changes this will need to include color: black; text-decoration: none;
+    &:active,
+    &:hover,
+    &:focus {
+      color: $topic_white;
+      text-decoration: underline;
+    }
   }
-}
\ No newline at end of file
+
+  .ama__link--blue,
+  %link--blue {
+    .topic__link--blue {
+      &:link,
+      &:visited {
+        color: $topic_blue;
+      }
+      // if the default link style changes this will need to include color: black; text-decoration: none;
+      &:active,
+      &:hover,
+      &:focus {
+        color: $topic_blue;
+        text-decoration: underline;
+      }
+    }
+  }
+}
```

Comment:
This nests the blue link inside the white link and makes it hard to extend the pattern. Also I'm not sure why the styles are nested inside of `.topic__link--blue` it seems like it should be a comma between `%link--blue` and `.topic__link--blue` instead of nesting them. The same comment is true for the white version.